egs/callhome_diarization/v1/diarization/make_rttm.py

```diff
@@ -1,73 +1,124 @@
-#!/bin/bash
+#!/usr/bin/env python
```

Comment:
@mmaciej2, this script looks a lot better now. Thanks! Could you change python to python3 in this line, and make sure it still works? Also, I think you can remove the from __future__  statement now.